include/drivers/timer/nrf_rtc_timer.h

```diff
@@ -11,8 +11,12 @@
 extern "C" {
 #endif
 
+/** @brief Maximum allowed time span that is considered to be in the future.
+ */
+#define NRF_RTC_TIMER_MAX_SCHEDULE_SPAN (BIT(23) - 1)
+
 typedef void (*z_nrf_rtc_timer_compare_handler_t)(int32_t id,
-						uint32_t cc_value,
+						uint64_t cc_value,
```

Comment:
Let's decide if we are going to provide `cc_value` or `current_time` at which we real triggered.
We are going to provide `current_time` to not have to read timer again in callback.
Let's name this parameter `current_time`, `curr_time` or `now_time` but not `cc_value`.